Feature/help contact form

nats12 opened the pull request:
- Created the contact form
- Minor accordion arrow change from using package css class to svgs
- Also decreased the accordion headings in size since latest master theme update

andrico1234 commented on src/components/accordion/accordionWrapper.js:
```diff
@@ -18,8 +22,14 @@ const AccordionWrapper = styled.div`
     border-radius: 4px;
     outline: none;
     background-color: ${props => props.theme.colors.lightGrey} !important;
```
you can combine background colour, image, position and repeat in a single rule!

nats12 replied:
Nice spot, changed!

andrico1234 commented on src/components/contactForm/dropDown.js:
```diff
@@ -0,0 +1,140 @@
+          isOpen={isOpen}
+          aria-expanded={isOpen}
+        >
+          {selectedItem != '' ? selectedItem : 'What is your question about?'}
```
double equals signs?

selectedItem is being defined as a string so it shouldn't change right?

also PropTypes please!

nats12 replied:
Double equal sign where?

selectedItem is defined as an empty string to begin with, then set to the drop down option’s ID as soon as one is selected from dropdown (still string). If selectedItem is empty, the drop down question text should appear, otherwise the selected option should appear on the drop down button. Does that make sense?

andrico1234 commented on src/components/contactForm/dropDown.js:
```diff
@@ -0,0 +1,140 @@
+  color: ${theme.colors.white};
+  overflow: hidden;
+  appearance: none;
+  background-image: url(${props =>
```
the 4 background rules can be simplified here as well

andrico1234 commented:
![image](https://user-images.githubusercontent.com/17087167/40935977-aa710a10-6831-11e8-8f07-0e080a271233.png)

I haven't seen the mockups for this page, so maybe there's supposed to be a hero image on the top of the page. But if not, then is the form and accordion supposed to be staggered like this? It might be the browser funking up your styles

andrico1234 commented:
also, getting this Prop-Type error

![image](https://user-images.githubusercontent.com/17087167/40936032-dcc81ecc-6831-11e8-97a6-d0c75557cee8.png)

andrico1234 commented:
![image](https://user-images.githubusercontent.com/17087167/40936048-e8063030-6831-11e8-87e3-fe5ed7afe62e.png)
the placeholder hovering over the input text

andrico1234 commented:
a little more trivial, but having a little animation when the user clicks on the different inputs on the form could make it feel a bit more fluid. Maybe have the placeholder text move up and shrink, as the green border grows a little bit!

nats12 commented:
Which browser/breakpoint were you checking the inputs on? As the label placeholder already shrinks, moves up and changes color to green but from the looks of your screen shot it doesn’t seem to be showing up for you?

nats12 commented:
As for the hero yeah the banner is meant to go above the accordion, will add that after this is merged in. As for overlapping form, I’ll double check break points again. It could be that I need to break the form sooner

andrico1234 commented:
![image](https://user-images.githubusercontent.com/17087167/41003382-882a8eb2-690e-11e8-97c4-6c1ceb445435.png)

currently running on Chrome. Also anything between 1024px and 1300px has the help form overlapping with the accordion

andrico1234 commented:
also it might be worth adding nav bar routing to the help page, should be a quick addition

nats12 commented:
@andrico1234 made the following changes:
- fixed the overlapping of accordion and form - i had placed the form outside the grid
- simplified background css rules
- added some animation on the labels when they're on focus
- fixed the button prop-type error
- changed comparison on dropdown.js to use the double equal sign 

As for using prop-types - my components don't receive any props so surely I don't need them?

As for menu - I don't think we have the functionality to be able to link to pages yet on master? Correct me if I'm wrong!

I can't reproduce the input text overlapping the placeholder on my end, so please review again after these fixes and let me know if that's still happening :)

laij84 commented on src/components/accordion/accordionWrapper.js:
```diff
@@ -1,12 +1,16 @@
     p {
+      font-size: 18px;
+      line-height: 27px;
       @media (max-width: ${props => props.theme.breakpoints[1]}) {
```
avoid max-width BPs, as technically this should be the breakpoint -1px. This will conflict with any min-width styles of the same BP, since they will overlap at that px size.

laij84 commented on src/components/contactForm/index.js:
```diff
@@ -0,0 +1,264 @@
+    margin: 0 -20px 0 -20px;
+  };
+
+  @media (min-width: ${props =>
```
Can you import and use the media functions from theme/media.js
